site/snippets/language-select.php

```diff
@@ -0,0 +1,11 @@
+<ul class="language-select">
+  <?php foreach($site->languages() as $language): ?>
+    <?php if ($language != $site->language()): ?>
```

Comment:
I think I want to change this. This only shows the _other_ language, so we see the German flag when viewing English content and vice-versa. That is confusing.

Instead of this, I want to show both options, but only make one of them selectable. That shows which one is currently selected and signals that the other one can be selected.